plugin/references.py

```diff
@@ -191,3 +191,19 @@ def _group_references_by_file(self, references: 'List[ReferenceDict]'
         linecache.clearcache()
 
         return grouped_references
+
+
+def get_line(file_name: str, row: int) -> str:
+    '''
+    Get the line from the buffer if the view is open, else fallback to linecache.
+    '''
+    view = sublime.active_window().find_open_file(file_name)
```

Comment:
I don't really like the `sublime.active_window()` call here. We already know what the view is in `_group_references_by_file`, it is `self.view`, and so we also know what the window is, namely `self.view.window()`. Perhaps `get_line` can get an extra parameter for the `window`?